Google.Api.Generator/Generation/ServiceMethodGenerator.cs

```diff
@@ -198,6 +209,8 @@ private partial class MethodDef
             private DocumentationCommentTriviaSyntax BidiStreamingSettingsXmlDoc => XmlDoc.Param(BidiStreamingSettingsParam, "If not null, applies streaming overrides to this RPC call.");
             private DocumentationCommentTriviaSyntax ReturnsBidiStreamingXmlDoc => XmlDoc.Returns("The client-server stream.");
             private DocumentationCommentTriviaSyntax ReturnsServerStreamingXmlDoc => XmlDoc.Returns("The server stream.");
+            private DocumentationCommentTriviaSyntax ClientStreamingSettingsXmlDoc => XmlDoc.Param(BidiStreamingSettingsParam, "If not null, applies streaming overrides to this RPC call.");
```

Comment:
BidiStreamingSettingsParam => ClientStreamingSettingsParam

It doesn't end up mattering because both parameters have the same name, but it's worth being consistent :)

Reply:
👍